project/MimaVersionPlugin.scala

```diff
@@ -86,13 +85,7 @@ object MimaVersionPlugin extends AutoPlugin {
       // Focusing on backward compat as opposed to forward, for now.
       ProblemFilters.exclude[ReversedMissingMethodProblem]("smithy4s.*"),
       ProblemFilters.exclude[Problem]("*.internals*"),
-      ProblemFilters.exclude[IncompatibleResultTypeProblem]("*fromSchema"),
-      ProblemFilters.exclude[DirectMissingMethodProblem](
```

Comment:
since we're able to break BC in 0.17 anyways. @Baccata can we drop the `*fromSchema` exclusion too?

Reply:
Sure

Reply:
done in 06bedfbb8092ebf0ce856e44f38a6d1f939e3271